StudCity.Frontend/src/services/authenticateService.js

```diff
@@ -68,10 +68,36 @@ export class AuthenticateService {
 			if(err.status === 401) {
 				showAlert("You are not autorized!", "error");
 				window.location = "/authenticate";
+				return false;
 			}
 			
 			showAlert(err.response.data.error, "error");
 			return false;
 		}
 	}
+
+	async forgotPassword(email) {
+		try {
+			const response = await this.service.get(`${this.authUrl}/forgot-password/${email}`);
```

Comment:
and if you only use status from this api call, then I would suggest write it like this:
const { status } = await ....